walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +274,158 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeCannotAttend() {
```

Comment:
Do these tests pass now? I thought there are TODOs?

Reply:
You're right. I added 5 tests for the optional attendees before writing the code so they have not been tested yet.

Reply:
Yeah we don't add test cases that are not implemented yet. We won't be able to submit failing tests due to Google's TapPresubmit checks.
I guess they all pass today?

Reply:
Oh ok. Yes these all pass now.